README.md

```diff
@@ -82,12 +82,16 @@ Fork or download the HTM-Community Nupic.cpp repository from https://github.com/
 The easiest way to build from source is as follows. 
 ```
     cd to-repository-root
-    python setup.py install --user --prefix=
+    python setup.py install --user --force
```

Comment:
do we eat our dog food? is this used in CI?

Reply:
Travis ci does not use this. I dont know about yhe other CIs.  I personally use these flags.